spec

```diff
@@ -957,7 +957,6 @@ endseg
 beginseg
     name "ovl_Bg_Ingate"
     include "build/src/overlays/actors/ovl_Bg_Ingate/z_bg_ingate.o"
-    include "build/data/overlays/actors/z_bg_ingate.data.o"
     include "build/data/overlays/actors/z_bg_ingate.reloc.o"
 endseg
```

Comment:
youll want to change this to use the auto generated reloc. at the moment this is using reloc from the original rom.
to do this you can replace the reloc line with 
include "build/src/overlays/actors/ovl_Bg_Ingate/ovl_Bg_Ingate_reloc.o"

Reply:
👍